src/utils/sourcemaps/inject.rs

```diff
@@ -481,6 +494,41 @@ something else
 
     #[test]
     fn test_normalize_sourcemap_url() {
+        // TODO: Enable the following test and make it pass
+        // Linux allows having `\` in a file name but our path helpers,
+        // specifically `join_path` and `clean_path` from `symbolic::common`,
+        // do not use `std::path::MAIN_SEPARATOR` and instead tries to guess
+        // the path style by the existence of a `\` in the path. This is
+        // problematic because it can lead to incorrect path normalization.
+        // assert_eq!(
+        //     normalize_sourcemap_url("/foo/ba\\r/baz.js", "baz.js.map"),
+        //     "/foo/ba\\r/baz.js.map"
+        // );
+
+        assert_eq!(
+            normalize_sourcemap_url(
+                &format!("foo{0}bar{0}baz.js", std::path::MAIN_SEPARATOR),
```

Comment:
TIL about this syntax

Reply:
Me too. This is my first working Rust patch 😂